src/enterprise/enterprise.proto

```diff
@@ -65,6 +85,9 @@ message GetActivationCodeResponse {
   string activation_code = 3;
 }
 
+message HeartbeatRequest{}
+message HeartbeatResponse{}
```

Comment:
This is documented on line 105:

```
// Heartbeat is used in testing to trigger a heartbeat on demand. Normally this happens
// on a timer.
```

The enterprise service `Heartbeat` RPC is only used in testing to trigger a heartbeat. The license service `Heartbeat` has the actual logic.

Reply:
Whoops :joy:. That makes sense!

I think it's taken me a while to internalize the enterprise API vs. the license API (which both have e.g. `Activate`, `Heartbeat`, etc, which makes sense the because the former is sort of a cache/client interface to the latter). Anyway, thanks for the clarification!